### PR TITLE
Feat/age7 configuration

### PR DESCRIPTION
**Pull request by julien-devatom**
### Description of change
- Following this [MIP](https://forum.morpho.org/t/mip-morpho-rewards-age-7/348)
- If the vote is not massing, we'll remove the age7 configuration from the main branch.

### Pull-Request Checklist

- [ ] Code is up-to-date with the `main` branch
- [ ] `yarn lint` passes with this change
- [ ] `yarn test` passes with this change
- [ ] This pull request links relevant issues as `Fixes #0000`
- [ ] There are new or updated unit tests validating the change
- [ ] Documentation has been updated to reflect this change
- [ ] The new commits follow conventions outlined in the [conventional commit spec](https://www.conventionalcommits.org/en/v1.0.0/)

### CODE REVIEW

File: src/age-epochs.json [hunk 1]
```diff
@@ -304,5 +313,66 @@
...
+        {
+          "symbol": "aDAI",
+          "market": "0x028171bCA77440897B824Ca71D1c56caC55b68A3",
+          "weight": 228
+        },
+        {
+          "symbol": "aWETH",
+          "market": "0x030bA81f1c18d280636F32af80b9AAd02Cf0854e",
+          "weight": 4690,
+          "sizeDistribution": {
```
Comment (oumar-fall) [inline src/age-epochs.json:335]:
```suggestion
          "sideDistribution": {
```

File: src/age-epochs.json [hunk 2]
```diff
@@ -304,5 +313,66 @@
...
+            "type": "static",
+            "params": {
+              "supplySize": 5000,
```
Comment (oumar-fall) [inline src/age-epochs.json:338]:
```suggestion
              "supplySide": 5000,
```

**Review by oumar-fall — APPROVED** [review]

**Comment by oumar-fall** [comment]
I didn't check the values though

**Review by mlgarchery — APPROVED** [review]

File: src/age-epochs.json [hunk 3]
```diff
@@ -294,11 +303,72 @@
...
+          "market": "0x9ff58f4fFB29fA2266Ab25e75e2A8b3503311656",
+          "weight": 774
+        },
+        {
+          "symbol": "cUSDT",
+          "market": "0xf650C3d88D12dB855b8bf7D11Be6C55A4e07dCC9",
+          "weight": 487
+        },
+        {
+          "symbol": "aWETHV3",
```
Comment (mlgarchery) [inline src/age-epochs.json:364]: it's aWETHv3 or aWETHV3 ?

File: src/ages/distributionScripts/simpleVoteDistribution.ts [hunk 4]
```diff
@@ -11,11 +11,11 @@ interface MarketWeight {
   symbol: string;
   market: string;
   weight: number;
-  sizeDistribution?: {
+  sideDistribution?: {
     type: string;
```
Comment (ghivert) [inline src/ages/distributionScripts/simpleVoteDistribution.ts:15]: Isn't it a union here ? `type: 'static'` ?

**Review by ghivert — CHANGES_REQUESTED** [review]
I really think the type should be refined and expanded in the future. Otherwise everything's fine, and the values are correct according to the MIP.

**Review by ghivert — APPROVED** [review]

**Pull request merged**